pkg/download/classic/download.go

```diff
@@ -282,12 +282,25 @@ func downloadAndUnmarshalConfig(client dtclient.Client, theApi api.API, value va
 	if values, ok := data[theApi.PropertyNameOfGetAllResponse]; ok {
 		var res []map[string]any
 		err := mapstructure.Decode(values, &res)
+		if theApi.ID == api.KeyUserActionsWeb {
+			res = filterResponses(res, value)
```

Comment:
Is there a reason this step is performed regardless of whether an error occurred or not?